src/Pate/Proof/Instances.hs

```diff
@@ -102,135 +92,48 @@ import qualified What4.Interface as W4
 import qualified What4.ExprHelpers as WEH
 import qualified What4.Expr.Builder as W4B
 import qualified What4.Expr.GroundEval as W4G
+import qualified What4.Partial as W4P
 
 data SomeProofSym (arch :: DK.Type) tp where
```

Comment:
How that there is no more `ProofSymNonceExpr`, is this maybe more of a `SomeProofNonceExpr`? Or is that name taken already?

Reply:
Yeah this makes sense